lib/commands/set-default-argv.js

```diff
@@ -98,9 +98,10 @@ module.exports = function (internals, isModule) {
       .describe('log-level', 'Set the log-level, for example sql|warn')
       .string('log-level');
   } else {
+    var _internalsArgv = Object.assign(defaultConfig, internals.cmdOptions);
```

Comment:
we stop using var from now on, since we dropped support prior node 4. Use const here instead.